app/controllers/chargeback_controller.rb

```diff
@@ -112,7 +112,7 @@ def cb_rate_edit
       session[:changed] =  false
       replace_right_cell
     when "save", "add"
-      id = params[:id] && params[:button] == "save" ? params[:id] : "new"
+      id = params[:button] == "save" ? params[:id] : "new"
       return unless load_edit("cbrate_edit__#{id}", "replace_cell__chargeback")
```

Comment:
That's ok. I keep setting the `id` variable. The problem was that 

(1) `nil && "x"` returns "x"
(2) `"a" && "x"` returns "x"

That means `params[:id] &&` needs to be purged.